### DISCUSSION TITLE
update problems

### DISCUSSION THREAD

**Discussion by nelsonepr** [AzuraCast/AzuraCast#7134] [General]
Greetings, for some time now I have been having problems when making updates, via the web it does not do it, and the process begins through the ssh console but it throws this error that I leave in the image, if you can help me I will be grateful.
![falla](https://github.com/AzuraCast/AzuraCast/assets/72674065/fd780626-3ef3-4b61-8773-4a637e706023)

**Comment by Heimatradio** [comment]
Your hardisc is full 
failed to register layer: disk quota exceeded

**Reply by nelsonepr** [reply]
my hard disk is 80GB and now have 61GB is no full

**Reply by LordHelmchen666** [reply]
`failed to register layer: disk quota exceeded`
Heimatradio is right. You have not enough space left!
What is the output of `df -h` ?

**Reply by nelsonepr** [reply]
![imagen](https://github.com/AzuraCast/AzuraCast/assets/72674065/b7111170-5982-4f17-81aa-0a8fbdfa9842)

As you can see I still have space available

**Reply by Heimatradio** [reply]
during an update you need more temporaly space on your drive, remove some Audio-files, do the update and upload the audio-files again

**Reply by LordHelmchen666** [reply]
It is like Heimatradio wrote. Not enough space left.

**Reply by nelsonepr** [reply]
You can see in the graph that I have 20GB, if that amount is not enough to do a routine update then we are wrong, a month ago I got the same error and I had to reinstall the azuracast from scratch, it worked fine and now that error repeats , the graph is a real test, do not invent it, I have another vps with just 20GB of storage and that error has not happened to me, that is why I am looking for expert help from this great forum.

**Reply by LordHelmchen666** [reply]
Maybe searching this error `failed to register layer: disk quota exceeded` would be helpful for your problem. 
https://www.google.com/search?client=firefox-b-e&q=failed+to+register+layer%3A+disk+quota+exceeded
It seems to be a Docker problem, not Azuracast.

in this forum they suggest to check disk quota https://forum.storj.io/t/disk-quota-exceeded-on-linux-with-docker/24831/10
Did you check quota for this user and group?
See also [debian - Disk quota exceeded problem - Unix & Linux Stack Exchange 28](https://unix.stackexchange.com/a/270107)

So try `df -i` to see if Inodes full

P.S. The most people here using Azuracast are the same expert as you are.

**Reply by nelsonepr** [reply]
Thanks for your help friend, I really did what you recommended, it's what I always do before coming to the forum, that is, searching on Google is the first thing I do.

p.s: I am not looking for help from users like you and me but from the experts that I have seen who know a lot and help in this community, I would be unable to help anyone if I do not have true knowledge.

**Reply by BusterNeece** [reply]
@nelsonepr Honestly, I don't have any more helpful information for you than what the community has already provided. While I consider myself an expert on AzuraCast itself (at least I'd certainly hope so), I don't know every possible error that Docker can throw at you, and I haven't encountered this one before. I'd be doing the same thing as everyone else, which is just searching the web for a solution.

**Reply by nelsonepr** [reply]
I do consider you an expert in azuracast, I have read every topic in which you have helped and I even have the impression that you are the developer of azura, imagine if you don't know how to solve this then I give up, will it be that I change VPS and migrate everything , thanks very much buddy.

**Reply by LordHelmchen666** [reply]
We are all willing to help if we can.
Could you post output of `df -i` or `df -hi` ? So we could see INodes usage. They could be limited on a VPS.

**Comment by nelsonepr** [comment]
![imagen](https://github.com/AzuraCast/AzuraCast/assets/72674065/b7111170-5982-4f17-81aa-0a8fbdfa9842)

**Reply by LordHelmchen666** [reply]
Ah sorry. df -i
-h shows space and -i shows inodes

**Comment by nelsonepr** [comment]
![imagen](https://github.com/AzuraCast/AzuraCast/assets/72674065/f07f6774-28bd-4b5d-9d74-826020f9efd8)

**Reply by LordHelmchen666** [reply]
Inodes enough free. So this is not your problem. Maybe a permission problem?

EDIT: Can you check quota? Maybe your docker user have exceeded allowed quota. 
Quota summary report all users : `repquota -a `
Report on all users over quota limits: `quota -q` 
Quota user1: `quota -u user1 `
